tests/modules/align_timestamps/test_align_timestamps_module.py

```diff
@@ -1,186 +1,133 @@
+            lfp_ts_entry, spikes_ts_entry
+        ]
+        input_probes.append(test_input_probe)
+
+        # Convert json test data expected results to *.npy file outputs
+        expected_lfp_path = test_dir_expected / f"{pname}_lfp_expected.npy"
+        expected_lfp = np.array(probe["aligned_lfp_timestamps"])
+        np.save(expected_lfp_path, expected_lfp)
+
+        expected_spk_path = test_dir_expected / f"{pname}_spikes_expected.npy"
+        expected_spk = np.array(probe["aligned_spike_timestamps"])
+        np.save(expected_spk_path, expected_spk)
+
+        expected_aligned.append({
+            "expected_aligned_lfp": str(expected_lfp_path),
+            "expected_aligned_spikes": str(expected_spk_path)
+        })
+
+    module_input = {
+        "sync_h5_path": str(
+            align_timestamps_res_dir
+            / test_case_name
+            / test_data["sync_file_fname"]
+        ),
+        "probes": input_probes
     }
 
+    return module_input, expected_aligned
 
-@pytest.fixture(scope="module")
-def run_align_timestamps_706875901(tmpdir_factory):
-    base_path = tmpdir_factory.mktemp("align_timestamps_integration")
-    executable = ["python", "-m", "allensdk.brain_observatory.ecephys.align_timestamps"]
-
-    input_json_path = os.path.join(base_path, "706875901_align_timestamps_input.json")
-    output_json_path = os.path.join(base_path, "706875901_align_timestamps_output.json")
-    executable.extend(["--input_json", input_json_path])
-    executable.extend(["--output_json", output_json_path])
-
-    input_json_template_path = os.path.join(
-        DATA_DIR, "706875901_align_timestamps_input.json"
-    )
-    apply_input_json_template(input_json_template_path, input_json_path, base_path)
-
-    sp.check_call(executable)
-
-    return output_json_path
-
-
-@pytest.mark.requires_bamboo
-def test_align_timestamps_parameters_706875901(
-    run_align_timestamps_706875901, align_timestamps_706875901_expected_params
-):
 
-    with open(run_align_timestamps_706875901, "r") as output_json_file:
-        output_json_data = json.load(output_json_file)
+@pytest.mark.parametrize("align_timestamps_test_case_fixture", [
+    {"test_case_name": "vcn_test_case_1"},
+    {"test_case_name": "vbn_test_case_1"}
 
-    for probe in output_json_data["probe_outputs"]:
-        expected = align_timestamps_706875901_expected_params[probe["name"]]
 
-        assert expected["total_time_shift"] == probe["total_time_shift"]
-        assert (
-            expected["global_probe_sampling_rate"]
-            == probe["global_probe_sampling_rate"]
-        )
-        assert (
-            expected["global_probe_lfp_sampling_rate"]
-            == probe["global_probe_lfp_sampling_rate"]
-        )
+], indirect=["align_timestamps_test_case_fixture"])
+def test_align_timestamps(align_timestamps_test_case_fixture):
+    test_input, test_expected = align_timestamps_test_case_fixture
 
+    print(test_input)
```

Comment:
Should these print statements be here?

Reply:
I'm generally okay with print statements in test code. Pytest suppresses any print statements such that they only show up when their associated test fails.